<?xml version="1.0"?>
<launch>
  <node name="tf_broadcaster" pkg="robot_tf"  type="tf_broadcaster" output="screen">
  <rosparam command="load" file="$(find robot_tf)/config/settings.yaml" /> 
  </node>

  <!--node name="odom_publisher" pkg="robot_tf"  type="odom_publisher" output="screen">
  </node-->

</launch>
